### DISCUSSION TITLE
How to show a heads up notification ?

### DISCUSSION THREAD

**Discussion by veermetri05** [invertase/react-native-firebase#4381] [Q&A] [answered]
I want to show a heads up notification
https://developer.android.com/guide/topics/ui/notifiers/notifications#Heads-up

And also show it on the lock screen. I didn't find any proper documentation about it.

**Comment by mikehardy** [comment, marked as answer]
Local device notifications are an API surface area that react-native-firebase does not cover

For that reason there will not be any proper or improper documentation about them here, really.

You'll want to integrate a full featured notification package and drive notification display through it, perhaps by using the notification package's APIs in the handlers you install for react-native-firebase data-only firebase cloud message receipt

Suggestions - react-native-notifications, notifee, react-native-push-notification as possible packages

**Reply by favaztechcurve** [reply]
how to handle this heads up in background and quit mode?

**Reply by hemaljoshi** [reply]
I am using notifee for forground and background notifications but if i am using it in quit mode there will be duplicate notifications how can i show heads up notifications
